source/application/AMPIC887DriverServer/AMPIC887AxisCollection.h

```diff
@@ -0,0 +1,77 @@
+#ifndef AMPIC887AXISCOLLECTION_H
+#define AMPIC887AXISCOLLECTION_H
+#include <QList>
+#include "AMGCS2.h"
+
+/*!
+  * A class representing a collection of axes controlled by a PI C887.11 Controller
+  */
+class AMPIC887AxisCollection : public QList<AMGCS2::Axis>
+{
+public:
+	// Enumerates the different states the collection can be initialized in.
+	enum InitializationState {
+		EmptyCollection,	// Collection is initialized with no contained axes
+		AllAxes,			// Collection is initialized with all axes
+		LinearAxes			// Collection is initialized with linear axes (X, Y, Z)
```

Comment:
Theoretically I could add this for completeness, but there are commands which require only linear axes (set pivot point) but no commands which require only rotational axes.